README.md

```diff
@@ -1,3 +1,59 @@
 # KubeSaw Docs
 
 This repository contains the documentation website code and Markdown source files for kubesaw.github.io.
+
+## Contributing
+
+The website is built using [mkdocs](https://www.mkdocs.org/getting-started/#getting-started-with-mkdocs) 
+
+### Prerequisites
+
+- python3
+```shell
+python3 --version
+Python 3.11.5
+```
+
+### 1.  Create an isolated virtual environment
+From the root folder of the repo run: 
+```shell
+python3 -m venv venv
+``` 
+NOTE: the second `venv` is the name of the folder containing the [virtual environment](https://docs.python.org/3/library/venv.html)
+
+Now you need to activate the virtual env so that all the dependencies will be installed only in here, and not on your global python intallation.
```

Comment:
```suggestion
Now you need to activate the virtual env so that all the dependencies will be installed only here, not on your global python installation.
```

Reply:
TY!